excavator/utils.py

```diff
@@ -80,7 +81,49 @@ def env_int(name, required=False, default=empty):
     value = get_env_value(name, required=required, default=default)
     if value is empty:
         raise ValueError(
-            "`env_int` requires either a default value to be specified, or, for "
+            "`env_int` requires either a default value to be specified, or for "
             "the variable to be present in the environment"
         )
     return int(value)
+
+
+def _parse_timestamp(timestamp_str):
+    timestamp = float(timestamp_str)
+    return datetime.datetime.fromtimestamp(timestamp)
+
+
+def _parse_iso8601(iso8601_str):
+    try:
+        import iso8601
+    except ImportError:
+        raise ImportError(
+            'Parsing iso8601 datetime strings requires the iso8601 library'
+        )
+    return iso8601.parse_date(iso8601_str)
+
+
+_formats = {
+    'timestamp': _parse_timestamp,
+    'iso8601': _parse_iso8601,
+}
+
+
+def env_datetime(name, default=empty, required=False, fmt='timestamp'):
```

Comment:
I'm convinced.  `env_timestamp/datetime/iso8601` it is.

Question, what is the behavior of `env_datetime?`  Does it try to use the `dateutils` library and yell loudly if it isn't installed?